app/app.go

```diff
@@ -486,6 +488,7 @@ func NewOsmosisApp(
 		lockuptypes.ModuleName,
 		crisistypes.ModuleName, govtypes.ModuleName, stakingtypes.ModuleName, claimtypes.ModuleName,
 		// Note: epochs' endblock should be "real" end of epochs, we keep epochs endblock at the end
+		gammtypes.ModuleName,
```

Comment:
Can you move the epochs comment below the gamm types line

Reply:
srry my bad!